modules/mint/minter.go

```diff
@@ -9,14 +9,13 @@ import (
 )
 
 const (
-	nanoToMiliSecond  = 1000000
-	miliSecondPerYear = 60 * 60 * 8766 * 1000
+	blocksPerYear = 60 * 60 * 8766 / 5 // 5 second a block, 8766 = 365.25 * 24
 )
```

Comment:
Considering the fact that the actual block-producing intervals are greater than 5 seconds (timeout_commit), how about we let blocksPerYear = 60*60*8766 / 6 ?